arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -386,99 +386,93 @@ static inline bool isStopping() {
   return application_features::ApplicationServer::isStopping();
 }
 
-arangodb::Result SynchronizeShard::getReadLock(std::string const& endpoint,
-                                               std::string const& database,
-                                               std::string const& collection,
-                                               std::string const& clientId, uint64_t rlid,
-                                               bool soft, double timeout) {
+arangodb::Result SynchronizeShard::getReadLock(
+  std::string const& endpoint, std::string const& database,
+  std::string const& collection, std::string const& clientId,
+  uint64_t rlid, bool soft, double timeout) {
+
+  // This function can be implemented in a more robust manner for server
```

Comment:
I think this comment is now missleading.
If i understand correctly we have now the "optimzied" version for 3.5 onwards, which can only
be used in conjunction with 3.4 or newer.